### PR TITLE
Increment Protocol Version to 18 to support Beta 1.9 pre-release

### PR DESCRIPTION
**Pull request by ghost**
Increment Version number to 18 to support Beta 1.9 pre-release.  Only
change is to protocolVersion in proto.go

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -10,3 +10,4 @@ mock_*.go
 deps.dot*
 diagrams/*.png
 diagrams/deps.dot
+.DS_Store
```
Comment (jnwhiteh) [inline .gitignore:13]: This doesn't belong in the main code repository. If you want to exclude .DS_Store files globally, then you can do something like this:

```
git config --global core.excludesfile ~/.gitexcludes
echo .DS_Store >> ~/.gitexcludes
```

There, now they're globally excluded.

**Comment by jnwhiteh** [comment]
At the very least this belongs in a branch. The master branch of chunkymonkey is designed to compile against the latest Go release, and work with the currently released version of Minecraft. Feel free to make the changes locally in a branch of your own repository.

File: .gitignore (hunk 1 above)
Comment (ghost) [inline .gitignore:13]: On Sat Sep 24 16:51:37 2011, jnwhiteh wrote:

> > @@ -10,3 +10,4 @@ mock__.go
> >  deps.dot_
> >  diagrams/*.png
> >  diagrams/deps.dot
> > +.DS_Store
> 
> This doesn't belong in the main code repository. If you want to exclude .DS_Store files globally, then you can do something like this:
> 
> git config --global core.excludesfile ~/.gitexcludes
> echo .DS_Store >> ~/.gitexcludes
> 
> There, now they're globally excluded.

Oh!  OK.  now that I have done this on my machine, do I need to change 
the pull request or can you just accept part of it?

**Comment by jnwhiteh** [comment]
I've closed the request outright since it is a trivial change for us to make once 1.9 is released, and huin is currently working on refactoring the connection/login code. The easiest thing for you to do locally is probably:

```
# make sure you're on your master branch
git checkout master
# branch to 1.9-pre, based on your current HEAD
git branch 1.9-pre
# revert master to the same HEAD as the main repo
git reset --hard c733defbc4983fa4bc15218d38bb6bb21c129c44
# push these changes to your fork on github
git push --all --force
```

You won't lose anything using this, you'll just move your current master/HEAD to pre-1.9/HEAD and you'll revert master/HEAD to match the main repository. In general, you should try to make any of your changes in a branch, ensuring that you still have a target you can use to sync against the main repository.

**Comment by huin** [comment]
I've grabbed the change as it is as the start of a branch for 1.9, but will remove the gitignore change.

This change doesn't hurt the connection/login changes, which are largely in the main module.

**Comment by jnwhiteh** [comment]
Sounds great, thanks!

**Pull request closed without merging**